tcms/testruns/tests/test_views.py

```diff
@@ -555,6 +555,17 @@ def setUpTestData(cls):
         cls.intern = UserFactory(username='intern',
                                  email='intern@example.com')
 
+    def test_should_be_able_to_open_testrun_edit_page(self):
+        response = self.client.get(self.edit_url)
+
+        self.assertEqual(HTTPStatus.OK, response.status_code)
+        self.assertNotContains(
```

Comment:
Also, why NotContains ? 

I think you are wrong here b/c the field with a populated value should be visible when opening the page for editting.

Reply:
This is my bad, yes it supposed to be assertContains.